drivers/adc/ad7606/ad7606.c

```diff
@@ -1069,26 +1069,26 @@ int32_t ad7606_set_ch_gain(struct ad7606_dev *dev, uint8_t ch,
  *                  0 - No errors encountered.
 *******************************************************************************/
 int32_t ad7606_set_config(struct ad7606_dev *dev,
-			  struct ad7606_config config)
+			  struct ad7606_config *config)
```

Comment:
i prefer the pass by value instead of pass by reference here because the structure is very small and also this is a function that's called at the initialization stage of the program, not in some loops/recursion, where repetitive stack copying could be a problem

pointers only make sense when structures are large and/or when structure could be optional (in this case it cannot be optional)

Reply:
fine by me: will drop;